Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationIdentification.cshtml.cs

```diff
@@ -150,6 +150,12 @@ public async Task<IActionResult> OnGet()
 
     public async Task<IActionResult> OnPost()
     {
+        // Ensure OrganisationScheme is valid
+        if (OrganisationScheme == null || OrganisationScheme.Count == 0)
+        {
+            ModelState.AddModelError(nameof(OrganisationScheme), "Please select your organisation type");         
```

Comment:
I know it was written before but we can avoid using "Please" in error messages.

Please refer following link... 
https://design-system.service.gov.uk/components/error-message/#:~:text=Be%20clear%20and%20concise

Reply:
@dpatel017 Thanks for the link. Looks like across application 'Please' has been used at several places and I have copied this message from Identifier screen in registration to consistency.

Reply:
Yes, please don't use 'please' in error messages. Can we go with 'Select your organisation type'?